### ISSUE TITLE
[develop] [camera widget] Video proxy

### ISSUE THREAD

**Issue by overload08** [domogik/domoweb#76]
I think it will be good to make a video proxy for camera widget.

If I am at home, it's ok but, if i am outside, i can't see the video...

**Comment by fritz-smh** [comment]
Yes, I agree, I have the same issue at home. For now I used apache to do the proxy but for commun users it is not a usable solution.
The proxy would also be usefull for url like http://login:password@ip/bla : this kind of url can't be displayed in chrome for img tags since release 30 for security reasons.

I don't really have an idea on how to implement/configure this.

If you or Ferllings have an idea it would help

**Comment by overload08** [comment]
Look at https://github.com/zferentz/maproxy
May be it can be ok

**Comment by overload08** [comment]
I can't help you in this wait, i tryed it but i'm very bad in python code... But maproxy is very small, tornado function, that's will be easy for you to implement it ;)

**Comment by fritz-smh** [comment]
I will try to look at this in the next weeks

**Comment by overload08** [comment]
Ok. I use SplitDNS for the moment...
